src/migrations/1661960517210-callouts-comments.ts

```diff
@@ -0,0 +1,35 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class calloutsComments1661960517210 implements MigrationInterface {
+  name = 'calloutsComments1661960517210';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE \`callout\` DROP CONSTRAINT \`FK_62ed316cda7b75735b20307b47e\``
+    );
+
+    await queryRunner.query(
+      `ALTER TABLE \`callout\` RENAME COLUMN discussionId TO commentsId;`
+    );
+
+    await queryRunner.query(
+      `ALTER TABLE \`callout\` ADD CONSTRAINT \`FK_62ed316cda7b75735b20307b47e\` FOREIGN KEY (\`commentsId\`) REFERENCES \`comments\`(\`id\`) ON DELETE SET NULL ON UPDATE NO ACTION`
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    // await queryRunner.query(
+    //   `ALTER TABLE \`callout\` DROP CONSTRAINT \`FK_62ed316cda7b75735b20307b47e\``
+    // );
+
+    // await queryRunner.query(
+    //   `ALTER TABLE \`callout\` RENAME COLUMN commentsId TO discussionId;`
+    // );
+
+    await queryRunner.query(`UPDATE \`callout\` SET discussionId = NULL;`);
```

Comment:
This is not going to revert the structure. Down should basically be a mirror of up.

Reply:
I need to set all these values to NULL because next sentence for adding the Foreign key is going to fail if I don't do it.
They were all null before these changes anyway. 
The only problem I see here is that, if we redeploy this changes after a rollback, we may leave callouts with type='discussion' with discussionId=NULL